streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -1845,6 +1845,28 @@ public void shouldSetAdminClientTimeout() {
         assertThat(assignorConfiguration.getAdminClientTimeout(), is(2 * 60 * 1000));
     }
 
+    @Test
+    public void shouldGetNextProbingRebalanceMs() {
+        nextProbingRebalanceMs.set(5 * 60 * 1000L);
+
+        createDefaultMockTaskManager();
+        final Map<String, Object> props = configProps();
+        final AssignorConfiguration assignorConfiguration = new AssignorConfiguration(props);
+
+        assertThat(assignorConfiguration.getNextProbingRebalanceMs(props).get(), equalTo(5 * 60 * 1000L));
+    }
+
+    @Test
+    public void shouldGetTime() {
+        time.setCurrentTimeMs(Long.MAX_VALUE);
+
+        createDefaultMockTaskManager();
+        final Map<String, Object> props = configProps();
+        final AssignorConfiguration assignorConfiguration = new AssignorConfiguration(props);
+
+        assertThat(assignorConfiguration.getTime(props).milliseconds(), equalTo(Long.MAX_VALUE));
+    }
+
```

Comment:
I was more thinking about a separate unit test class `AssignorConfigurationTest` with a more thorough testing of all code paths, but if we feel that this should also be a follow up task, that is also fine with me.

Reply:
Thanks for pointing these out, @cadonna .